src/CUITe/Controls/ControlBase.cs

```diff
@@ -124,11 +134,38 @@ public void WaitForControlReady()
         }
 
         /// <summary>
+        /// Waits for the control to be ready.
+        /// </summary>
+        protected void WaitForControlReadyIfNecessary()
+        {
+            if (isControlReadinessAwaitedByDefault)
+            {
+                sourceControl.WaitForControlReady();
+            }
+        }
+
+        /// <summary>
+        /// Waits for the control to appear in the user interface.
+        /// </summary>
+        public void WaitForControlExist()
+        {
+            sourceControl.WaitForControlExist();
```

Comment:
Return the value returned from `sourceControl.WaitForControlExist();`
```
/// <returns><code>true</code> if this control exists before the time-out; otherwise, <code>false</code>.</returns>
public int WaitForControlExist()
{
    return sourceControl.WaitForControlExist();
}
```